src/app/gallery/page.js

```diff
@@ -45,7 +53,7 @@ const Page = () => {
         </div>
       </div>
 
-      <div className="mb-14">
+      <div className={`${inView && "animate-flip-up animate-once"} mb-14`}>
```

Comment:
you may want to add another ref for this section as the user won't see the animation